src/main.ts

```diff
@@ -10,7 +10,7 @@ const bootstrap = async (): Promise<void> => {
     new ValidationPipe({
       //disableErrorMessages: true,
       whitelist: true,
-      forbidNonWhitelisted: true,
+      forbidNonWhitelisted: false,
```

Comment:
W jakim celu została wyłączona whitelista? Jest to globalne ustawienie i w znacznym stopniu odpowiada za bezpieczeństwo przyjmowanych danych od użytkownika. Nie może to być wyłączone!

Reply:
Whitelista nie została wyłączona, jedynie reguła wyrzucająca błąd gdy dane się nie zgadzają. Zamiast tego zostają obcięte jedynie do dozwolonych pól. Wymagało tego query filtrowania, które wyrzucało błąd w momencie w którym użytkownik odwoła się do nieprzewidzianych parametrów, co nie koniecznie jest czymś czego można by oczekiwać. Jeśli zakładamy, żea sypać błędami w przypadku dodatkowych danych,  opcja może zostać zmieniona na true.